### PR TITLE
Adds support for multiple config file Closes #17

### LINKED ISSUE #17: Search for a .ron folder locally and in $HOME
This should add support to search the current then parent directories for any .ron/ folders with *.yaml files, along with the current users ~/.ron folder. Any *.yaml files found would be added to the target list. Any name collisions would have to either add a suffix like a number or some other distinguishing factor.

For example, if there is a file ./.ron/ron.yaml and another in ~/.ron/ron.yaml, the command should make the local ron.yaml file `ron` and the home directory version `ron1`. It's either that or we have the local ron over ride the home ron file for the target.

### PR DESCRIPTION
**Pull request by pkar**
ron will attempt to find any additional files in the following order:

1. A -yaml flag filepath config file
2. A local ./ron.yaml or in a parent directory
3. Any files in the folder ./.ron/*.yaml
4. Any ~/.ron/*yaml files
5. A -default configuration or any binary built in targets

### CODE REVIEW

File: target/util_test.go [hunk 1]
```diff
@@ -49,3 +54,18 @@ func Test_keyIn(t *testing.T) {
 		})
 	}
 }
+
+func Test_homeDir(t *testing.T) {
+	if runtime.GOOS == "linux" {
```
Comment (boralyl) [inline target/util_test.go]: Should also add a test case or 2 for when the go OS is Windows.

**Review by boralyl — APPROVED** [review]

File: doc.go [hunk 2]
```diff
@@ -187,6 +187,14 @@ last.
 	$ ron target ron:prep
 	$ ron target prep
 
+ron will attempt to find any additional files in the following order:
+
+	1. A -yaml flag filepath config file
+	2. A local ./ron.yaml or in a parent directory
+	3. Any files in the folder ./.ron/*.yaml
```
Comment (noonat) [inline doc.go:194]: Just a drive by thought while reading this -- if we ever want to put other things in ~/.ron in the future we can put them in subfolders so they don't interfere with this glob loading.

Comment (pkar) [reply doc.go:194]: I don't have any ideas on a use case for another type of yaml config. If a special case does come up for ~/.ron a subfolder seems fine with me

**Review by boralyl — COMMENTED** [review]
Need to fix the build with the latest commit.  Just kidding, github PRs are very different from what I'm used to.

**Comment by pkar** [comment]
Going to merge this and work out any remaining bug fixes in separate PR's

**Pull request merged**